scripts/release_signal.py

```diff
@@ -0,0 +1,10 @@
+import os
+from subprocess import CalledProcessError, check_output
+
+try:
+    check_output('git describe --exact-match HEAD'.split())
+    print("We are in a tagged branch, let's do the release")
+except CalledProcessError as e:
+    print("We are not in a tagged branch")
+else:
```

Comment:
Why an "else" instead of putting it inside the try block?  Is this a way to be explicit?